docs/from_bosh_to_kube.md

```diff
@@ -357,13 +356,38 @@ tags:
 
 ## BPM
 
-### Entrypoint
+The BPM's are converted into containers of `ExtendedStatefulSet's` and `ExtendedJob's` pods. Each `process` in the BPM is run in a single `container` of kubernetes pods. Following subsections will describe about the mapping of BPM configuration into containers.
 
-### Environment
+### Entrypoint & Environment Variables
+
+Bosh             | Kube Pod Container |
+--------------- | ---- |
+`executable`        | `command` |
+`args`              | `args` |
+`env`              | `env` |
 
 ### Resources
 
-### Healthchecks
+Bosh             | Kube Pod Container |
+--------------- | ---- |
+`workdir`             | `workingDir`. Not Supported. |
+`hooks`               | `Not Supported` |
+`capabilities`        | `container.SecurityContext.Capabilities`. Not yet supported.  |
+`limits`              | `container.Resources.Limits`. Not yet supported. |
+`ephermeral_disk`     | Not Supported. |
+`persistent_disk`     | Supported. |
+`additional_volumes`  | maps to `VolumeClaimTemplates` which creates Persistent Volume claims in kube cluster. Not yet supported. |
+`unsafe`              | Unmapped to Kubernetes resources. Can be programmatically supported. Not yet supported. |
+
+## Healthchecks
```

Comment:
Couldn't find Healthcheck in BPM?

```suggestion
## Healthchecks

BPM doesn't provide information for health checks and relies on monit instead. CF-Operator provides health checks via the [BOSHContainerization](https://github.com/cloudfoundry-incubator/cf-operator/blob/2cab6516e8a3fc1f0c814554912449b88dee3101/pkg/bosh/manifest/containerization.go#L7-L13) property in the deployment manifest.
```

Reply:
Yes. As healthcheck is not part of BPM, It is not kept as a subsection under BPM. It is a separate section.